website/content/docs/k8s/connect/connect-ca-provider.mdx

```diff
@@ -14,173 +14,8 @@ Consul has support for different certificate authority (CA) providers to be used
 Please see [Connect Certificate Management](/docs/connect/ca) for the information on the providers
 we currently support.
 
-Generally, to configure a provider via the Consul Helm chart, you need to follow three steps:
+To configure a provider via the Consul Helm chart, you need to follow three steps:
 
 1. Create a configuration file containing your provider information.
 1. Create a Kubernetes secret containing the configuration file.
 1. Reference the Kubernetes secret in the [`server.extraVolumes`](/docs/k8s/helm#v-server-extravolumes) value in the Helm chart.
-
-Below we will go over this process for configuring Vault as the Connect CA.
-However, other providers can be configured similarly by providing the appropriate `ca_config`
-and `ca_provider` values for the provider you're using.
-
```

Comment:
Add a Header here for Vault as a Secrets Backend (Consul K8s 0.38.0 and higher)

For more information on using Vault as a secrets backend go see docs/k8s/installation/vault